hw/ip/rbox/data/rbox.hjson

```diff
@@ -0,0 +1,576 @@
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      regwen: "cfg_wen_ctl",
+      resval: "2",
+      fields: [
+        { bits: "0",
+	  name: "bat_disable_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "1",
+	  name: "ec_rst_l_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "2",
+	  name: "pwrb_out_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "3",
+	  name: "key0_out_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "4",
+	  name: "key1_out_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "5",
+	  name: "key2_out_0",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "6",
+	  name: "bat_disable_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "7",
+	  name: "ec_rst_l_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "8",
+	  name: "pwrb_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "9",
+	  name: "key0_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "10",
+	  name: "key1_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "11",
+	  name: "key2_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+      ]
+    }
+    { name: "pin_out_ctl",
+      desc: "configure the pin out directly to override the function",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "2",
+      fields: [
+        { bits: "0",
+	  name: "bat_disable",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "1",
+	  name: "ec_rst_l",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "2",
+	  name: "pwrb_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "3",
+	  name: "key0_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "4",
+	  name: "key1_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "5",
+	  name: "key2_out",
+	  desc: "0: disable override;1: enable override",
+	}
+      ]
+    }
+    { name: "pin_out_value",
+      desc: "confiure the pin out value directly to override the function",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "0",
+      fields: [
+        { bits: "0",
+	  name: "bat_disable",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "1",
+	  name: "ec_rst_l",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "2",
+	  name: "pwrb_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "3",
+	  name: "key0_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "4",
+	  name: "key1_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "5",
+	  name: "key2_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+      ]
+    }
+    { name: "pin_in_value",
+      desc: "For SW to read the rbox inputs like GPIO",
+      swaccess: "ro",
+      hwaccess: "hrw",
```

Comment:
does hardware need read access for this stuff?

Reply:
No, HW does not need the read access. Will fix.